runtime/vm/ArrayCopyHelpers.cpp

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2024
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+#include "vm_api.h"
+#include "vm_internal.h"
+
+#include "ArrayCopyHelpers.hpp"
+#include "VMAccess.hpp"
+
+extern "C" {
+
+/**
+ * @brief The helper method performs a memcpy from an object array to the native array.
+ *
+ * @param currentThread the thread performing the arraycopy
+ * @param arrayObject the object array to copy from
+ * @param isCopy a flag denoting the copy is completed upon return
+ * @param ensureMem32 a flag specific to the zOS/31bit
+ *
+ * @return the native array with the copied content
+ */
```

Comment:
Please don't repeat comments now found in `vm_internal.h`.

Reply:
Removed as suggested above.